android/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Replace com.test.game with the identifier of your game below, e.g.
+     com.gamemaker.game
+-->
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="org.touch.hle"
+    android:versionCode="1"
+    android:versionName="1.0"
+    android:installLocation="auto">
+
+    <!-- OpenGL ES 2.0 -->
+    <uses-feature android:glEsVersion="0x00020000" />
```

Comment:
I guess it doesn't matter, but this is wrong, right? :)

Reply:
yeap, the whole SDL project was copy-pasted (as advised for "complex" projects), i didn't saw that one

Removed